src/tagging/components/valueModifier.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Col, ControlLabel, FormGroup } from 'patternfly-react';
+import ValueSelector from './valueSelector';
+
+const ValueModifier = ({
+  tagValues, onTagValueChange, selectedTagValue, multiValue, valueLabel,
+}) => {
+  console.log('val mod:', tagValues, onTagValueChange, selectedTagValue, multiValue, valueLabel);
+  return (
+    <FormGroup>
+      <Col xs={12} md={4} lg={6}>
+        <ControlLabel>{valueLabel}</ControlLabel>
+      </Col>
+      <Col xs={12} md={8} lg={6}>
+        <ValueSelector tagValues={tagValues} onTagValueChange={onTagValueChange} selectedOption={selectedTagValue} multiValue={multiValue} />
+      </Col>
+    </FormGroup>
+  );
+};
+
+ValueModifier.propTypes = {
+  selectedTagValue: PropTypes.object.isRequired,
+  onTagValueChange: PropTypes.func.isRequired,
+  valueLabel: PropTypes.string,
+  multiValue: PropTypes.bool,
+  tagValues: PropTypes.arrayOf(PropTypes.object),
+};
+
+ValueModifier.defaultProps = {
+  valueLabel: 'Assigned Value:',
```

Comment:
I think this can be "Value" (drop assigned).  Also I'd remove the colon if this is for future product work - if it's for current ManageIQ I'd see keeping it for consistency.  If you remove it, make sure you remove it from Category as well.

Reply:
Done.